dsd/vienna_nupack.py

```diff
@@ -19,13 +19,21 @@
 from multiprocessing.pool import ThreadPool
 from typing import Sequence, Union, Tuple, List, Dict, Iterable, Optional
 
+try:
```

Comment:
The more elegant way to do this is to do the import within the function that needs it. Then if they don't have nupack Python bindings installed, the import fails if and only if they call the function that needs it. 

The current way, I'm not even sure what happens if the import fails and then they call the function.

Reply:
Will do, I think I will follow the same pattern as used for the `psutil` import for `cpu_count` function